cli/azd/pkg/input/console.go

```diff
@@ -509,6 +510,10 @@ func (c *AskerConsole) Confirm(ctx context.Context, options ConsoleOptions) (boo
 
 // wait until the next enter
 func (c *AskerConsole) WaitForEnter() {
+	if c.noPrompt {
+		return
+	}
```

Comment:
I'm assuming internally the device code auth will just poll to check if the code has been authenticated?

Reply:
Yes.
azd is prompt is just to hold the screen-switching (from terminal to browser) so the user can read and copy-paste the code